examples/download_version.rs

```diff
@@ -1,12 +1,98 @@
-use std::env;
+use std::{
+    env,
+    io::Stdout,
+    sync::{
+        atomic::{AtomicU64, Ordering},
+        Arc, Mutex,
+    },
+};
 
-use mc_downloader::ClientDownloader;
+use mc_downloader::prelude::{ClientDownloader, DownloadVersion, Reporter};
+use pbr::ProgressBar;
+
+struct ProgressTrack {
+    curr: AtomicU64,
+    total: AtomicU64,
+    pb: Mutex<ProgressBar<Stdout>>,
+}
 
 fn main() {
-    let dir = env::current_dir().unwrap();
-    let target = format!("{}{}", dir.to_str().unwrap().to_string(), "\\.minecraft");
-    let downloader: ClientDownloader = ClientDownloader::new();
-    downloader
-        .download_version("1.19.3".to_string(), target)
-        .unwrap();
+    println!("Start Minecraft Downloader");
+    let mut dir = env::current_dir().unwrap();
```

Comment:
I think using the current directory is not the right approach. You can use something like the [directories](https://lib.rs/crates/directories) crate to get the right path.

Reply:
I think that for this case is totally viable, in fact it is based on the original code, since it is only an example of how to use the library is not really oriented to a real use and is not expected that this script is executed as a main resource, even so if you want I can accommodate this